eclipse-scout-core/src/util/MoveSupport.less

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+#scout {
+
+  .move-support() {
+
+    &.dragged {
+      z-index: -1; // draw behind all other elements (during animation)
+      border: 0 !important; // remove border (if there is one) while dragging so the overlay covers the whole element
+
+      &.releasing {
+        opacity: 0;
+        transition: opacity 500ms;
+
+        &.moved {
+          transition: none;
+        }
+      }
+
+      // Overlay over currently dragged element
+      &::after {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        border-radius: inherit;
+        background-color: @move-support-drag-placeholder-background-color;
+        border: 2px dashed @move-support-drag-placeholder-border-color;
+        z-index: 2; // draw over elements inside the tile having a z-index
+      }
+
+      & .scroll-shadow {
+        // Scroll shadow would be drawn over after element -> hide it
+        --scroll-shadow-size: transparent !important;
+        transition: none;
+      }
+    }
+
+    // done class is added when dragging is finished and original state restored right before dragged and dragged-end classes are removed
+    // the class will be removed about 50ms later and is used to disable the opacity transition. This ensures the dragged element does not flicker when shown again.
+    &.drag-done {
+      transition: none !important;
+    }
+
+    &.clone.dragging {
```

Comment:
I found the name dragging confusing and too generic, that is why I added the clone class. I replaced it now with dragging-clone, since we do not need both classes.